src/components/AppNavigation/AppNavigation.vue

```diff
@@ -85,6 +90,23 @@ export default {
 
 	mixins: [isMobile],
 
+	props: {
+		/**
+		 * aria-label for the app navigation element
+		 */
+		ariaLabel: {
+			type: String,
+			default: '',
+		},
+		/**
+		 * aria-label for the toggle button
+		 */
+		toggleAriaLabel: {
+			type: String,
+			default: '',
+		},
```

Comment:
Do we really need them?

```suggestion
		ariaLabel: {
			type: String,
			default: t('App navigation'),
		},
		/**
		 * aria-label for the toggle button
		 */
		toggleAriaLabel: {
			type: String,
			default: t('App navigation toggle'),
		},
```

Reply:
Seems it's possible to set the aria-label anyway from outside, so probably not needed as property.

With the toggle I have mixed feelings as it's currently not reachable with keyboard nav, but we might need to do it.

We could also agree to settle on "App navigation" and "App navigation toggle" for these names regardless of the app.

The only thing is that with the talk app it doesn't feel like an app navigation as it's the conversation list, hence the idea of having a custom text there.

Reply:
I'm fine with having this if you think it's easier, but default to an empty string and fallback in the template is not the way to go :)
props defaults are here for a reason ;)

Reply:
> Seems it's possible to set the aria-label anyway from outside, so probably not needed as property.

Only for the top container though, not for the toggle

Reply:
ah ok.

anyway, I'll certainly return to this once I look into why we cannot tab to the toggle